__tests__/api.test.js

```diff
@@ -69,63 +69,107 @@ describe("GET API - all reviews", () => {
 describe("POST API - insert a comment row using review_id and author FK", () => {
     test("201: sucessfully insert a new comment in the comments table", () => {
         const dummyData = {
-            "author": "bainesface",
-            "body": "This is some body text, hi2",
-            "review_id": 2
+            "username": "bainesface",
+            "body": "This is some body text, hi2"
         }      
         return request(app)
             .post("/api/reviews/2/comments")
             .send(dummyData)
             .expect(201)
             .then(({body}) => {
-                expect(body).toEqual({status:'Inserted successfully', rowCount:1});
+                const expected = {
+                    "status": "Inserted successfully",
+                    "rowCount": 1,
+                    "comment": {
+                        "username": "bainesface",
+                        "body": "This is some body text, hi2"
+                    }
+                };
+                expect(body).toEqual(expected);
+            });
+    });
+    test("400: invalid extra properties", () => {
```

Comment:
if were are testing for extra we would want all the original keys so username > fakeUsername